assignments/objects.js

```diff
@@ -18,31 +18,78 @@ let example = {
 }
 
 // Write your intern objects here:
+let student1 = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "melloy@psu.edu"
+  "gender": "F"
+}
+
+let student2 = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kmummery2@wikimedia.org"
+  "gender": "M"
+}
+
+let student3 = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org"
+  "gender": "M"
+}
+
+let student4 = {
+  "id": 5,
+  "name": "Gannie",
+  "email": "gmartinson@illinois.edu"
+  "gender": "M"
+}
 
+let student5 = {
+  "id": 6,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com"
+  "gender": "F"
+}
 
-// ==== Challenge 2: Reading Object Data ==== 
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(student1.name);
 // Kennan's ID
+console.log(student2.id);
 
 // Keven's email
+console.log(student3.email);
 
 // Gannie's name
+console.log(student4.name);
 
 // Antonietta's Gender
+console.log(student5.gender;
+
 
-// ==== Challenge 3: Object Methods ==== 
+// ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+  function 
+
+
+
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
+  function antonietta (x, y) {
```

Comment:
So here, we want to add a method to the object that refers to Antonietta, in this case you have her as `student5`.

Sort of like how `student5` has a `gender` property, we can add functions in the same way. Here, we want to add a function to `student5`

This can be done just like assigning something to a variable:
```
student5.multiplyNums = function(x, y) {
   return x * y;
}
```

This way you can call `student5.multiplyNums(4, 6)` just like any other function.